orc8r/cloud/helm/orc8r/charts/nms/values.yaml

```diff
@@ -25,16 +25,16 @@ rbac: false
 magmalte:
   create: true
   env:
-    api_host: orc8r-nginx-proxy.magma.svc.cluster.local
+    api_host: orc8r-nginx-proxy
     host: 0.0.0.0
     port: 8081
     mapbox_access_token: ""
-    mysql_host: mariadb.magma.svc.cluster.local
-    mysql_port: 5432
+    mysql_host: mysql
+    mysql_port: 3306
     mysql_db: magma
     mysql_user: magma
     mysql_pass: password
-    mysql_dialect: postgres
+    mysql_dialect: mysql
```

Comment:
cc @andreilee for whether this should be Postgres at this point?

Reply:
Same https://github.com/magma/magma/pull/6413#discussion_r740730205